### PR TITLE
updated to beta.3 and fixed a few issues

### PR DESCRIPTION
**Pull request by cammace**

### CODE REVIEW

File: MapboxAndroidDemo/build.gradle [hunk 1]
```diff
@@ -1,18 +1,19 @@
...
   buildToolsVersion "25.0.2"
 
   defaultConfig {
+    applicationId "com.mapbox.mapboxandroiddemo"
     minSdkVersion 15
     targetSdkVersion 25
-    versionCode 37
-    versionName "4.2.2"
+    versionCode 38
+    versionName "5.0.0-beta.1"
```
Comment (Guardiola31337) [inline MapboxAndroidDemo/build.gradle]: Shouldn't be `5.0.0-beta.3`?

File: MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/location/AnimatedLocationIconActivity.java [hunk 2]
```diff
@@ -193,6 +194,7 @@ private void enableLocation() {
     locationListener = new LocationEngineListener() {
       @Override
       public void onConnected() {
+
```
Comment (Guardiola31337) [inline MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/location/AnimatedLocationIconActivity.java]: Blank line.

File: MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/styles/AddWmsSourceActivity.java [hunk 3]
```diff
@@ -44,7 +44,7 @@ public void onMapReady(MapboxMap mapboxMap) {
 
         // Add the web map source to the map.
         RasterLayer webMapLayer = new RasterLayer("web-map-layer", "web-map-source");
-        mapboxMap.addLayer(webMapLayer, "aeroway-taxiway");
+        mapboxMap.addLayerBelow(webMapLayer, "aeroway-taxiway");
```
Comment (Guardiola31337) [inline MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/styles/AddWmsSourceActivity.java]: Blank line.

File: MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/styles/CreateHeatmapPointsActivity.java [hunk 4]
```diff
@@ -131,7 +131,7 @@ private void addClusteredGeoJsonSource(MapboxMap mapboxMap) {
     unclustered.setFilter(
       neq("cluster", true)
     );
-    mapboxMap.addLayer(unclustered, "building");
+    mapboxMap.addLayerBelow(unclustered, "building");
 
     for (int i = 0; i < layers.length; i++) {
```
Comment (Guardiola31337) [inline MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/styles/CreateHeatmapPointsActivity.java]: Blank line.

File: MapboxAndroidWearDemo/build.gradle [hunk 5]
```diff
@@ -6,11 +6,11 @@ android {
...
   defaultConfig {
-    applicationId "com.mapbox.mapboxandroidweardemo"
-    minSdkVersion 21
+    applicationId "com.mapbox.mapboxandroiddemo"
+    minSdkVersion 23
     targetSdkVersion 25
-    versionCode 2
-    versionName "5.0.0"
+    versionCode 1
+    versionName "5.0.0-beta.1"
```
Comment (Guardiola31337) [inline MapboxAndroidWearDemo/build.gradle]: Shouldn't be 5.0.0-beta.3?

File: MapboxAndroidWearDemo/build.gradle [hunk 6]
```diff
@@ -6,11 +6,11 @@ android {
...
 
   defaultConfig {
-    applicationId "com.mapbox.mapboxandroidweardemo"
-    minSdkVersion 21
+    applicationId "com.mapbox.mapboxandroiddemo"
+    minSdkVersion 23
     targetSdkVersion 25
-    versionCode 2
-    versionName "5.0.0"
+    versionCode 1
```
Comment (Guardiola31337) [inline MapboxAndroidWearDemo/build.gradle]: Is it working? I think it has to be same `versionCode` as in _mobile_ module.

File: MapboxAndroidWearDemo/src/main/java/com/mapbox/mapboxandroidweardemo/examples/MapFragmentActivity.java [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+public class MapFragmentActivity extends WearableActivity {
+
+  @Override
+  protected void onCreate(Bundle savedInstanceState) {
+    super.onCreate(savedInstanceState);
+    setContentView(R.layout.activity_basic_support_map_frag);
+
+    LocationEngine locationEngine = LocationSource.getLocationEngine(this);
+    locationEngine.activate();
+
```
Comment (Guardiola31337) [inline MapboxAndroidWearDemo/src/main/java/com/mapbox/mapboxandroidweardemo/examples/MapFragmentActivity.java]: Blank lines.

File: MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/styles/CreateHeatmapPointsActivity.java (hunk 4 above)
Comment (cammace) [inline MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/styles/CreateHeatmapPointsActivity.java]: the blank line is intended to break up the different sections of code here.

Comment (Guardiola31337) [inline MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/styles/CreateHeatmapPointsActivity.java]: https://github.com/mapbox/mapbox-android-demo/blob/47a3ae460eadd3a85369371f7144b563a8e1bc9b/MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/styles/CreateHeatmapPointsActivity.java#L137

**Comment by tobrun** [comment]
Can we update this PR to 5.0.0 instead of 5.0.0.beta.3?

**Pull request merged**